src/fetchtastic/menu_app.py

```
@@ -0,0 +1,100 @@
+
+def run_menu() -> dict[str, list[str]] | None:
+    """Show one asset selector for Android APKs and Desktop installers."""
+    try:
+        apk_assets = menu_apk.fetch_apk_assets()
+    except Exception as exc:
+        logger.warning("Unable to fetch Android APK assets: %s", exc)
+        print(f"Warning: unable to fetch Android APK assets: {exc}")
+        apk_assets = []
+    try:
+        desktop_assets = menu_desktop.fetch_desktop_assets()
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)